oslo_vmware/rw_handles.py

```diff
@@ -22,6 +22,8 @@
 """
 
 import logging
+import socket
+
```

Comment:
I don't think this should add a newline and thus a new block of imports.

Reply:
Fixed. It was added like this by the IDE.